src/test/scala/com/mozilla/telemetry/streaming/EventsToAmplitudeTest.scala

```diff
@@ -87,8 +87,27 @@ class EventsToAmplitudeTest extends FlatSpec with Matchers with BeforeAndAfterAl
   // specific events we expect to see
   private def eventsJson(eventGroup: String) =
     s"""{ "event_type": "$eventGroup - AppOpen" }""" ::
-    s"""{ "event_type": "$eventGroup - Erase", "event_properties": { "erase_object": "erase_home" }, "user_properties": { "host": "side" }}""" ::
-    s"""{ "event_type": "second_event_group - AppClose", "event_properties": { "session_length": "1000" }}""" :: Nil
+    s"""
+      |{
+      |   "user_properties" : {
+      |      "host" : "side"
+      |   },
+      |   "event_properties" : {
+      |      "literal_field" : "literal value",
```

Comment:
This line is the only addition. Otherwise, these strings should be equivalent to the single-line versions modulo whitespace.